.gitignore

```diff
@@ -2,8 +2,10 @@
 *.sublime-project
 *.sublime-workspace
 *.DS_Store
+.idea/
 .temp
 .cache
+package-lock.json
```

Comment:
this no
it's often removed because of the meta repo / npm workspaces but ideally we would need package-lock.json

Reply:
Ok! Got it